Jenkinsfile

```diff
@@ -1,14 +1,29 @@
 pipeline {
     agent { label 'maven' }
     stages {
+        stage('Prepare') {
+            steps {
+                sh 'printenv'
+            }
+        }
         stage('Build') {
+            when {
+                expression { env.CHANGE_ID != null } // Pull request
+            }
             steps {
                 sh 'mvn -B -V clean verify -Prun-its -Pci'
             }
         }
+        stage('Deploy') {
+            when { branch 'master' }
+            steps {
+                echo "Deploy"
+                sh 'mvn help:effective-settings -B -V clean deploy -e -s ~/sonatype/settings.xml'
+            }
+        }
     }
     post {
-        always {
+        success {
             archiveArtifacts artifacts: 'deployments/launcher/target/*-complete.tar.gz', fingerprint: true
         }
```

Comment:
Do we need to post the skinny.tar as well ? Since we had update the Dockerfile to user that one and all the config is from configMaps/secrets.

Reply:
Of course. It seems we don't need the *complete.tar.gz any more. Am I right?

Reply:
I guess so, but let's get confirm from team.:)